db/post_migrate/20220613073844_unescape_event_name.rb

```diff
@@ -3,10 +3,14 @@
 class UnescapeEventName < ActiveRecord::Migration[7.0]
   disable_ddl_transaction!
 
+  TEMP_INDEX_NAME = "_temp_discourse_calendar_unescape_event_name_migration"
+
   def up
     # event notifications
-    start = 1
-    limit = DB.query_single("SELECT MAX(id) FROM notifications WHERE notification_type IN (27, 28)").first.to_i
+    DB.exec("CREATE INDEX CONCURRENTLY #{TEMP_INDEX_NAME} ON notifications(id) WHERE notification_type IN (27, 28)")
```

Comment:
Should this have `UnescapeEventName::` prefix too like the drop?

Reply:
Also why `ON notifications(id)` instead of just `ON notifications`?

Reply:
Oops nope, not necessary to have the prefix.

Reply:
Oh wait NVM I see!